spec/models/miq_ae_method_spec.rb

```diff
@@ -50,6 +50,44 @@
     expect { f1.valid? }.not_to make_database_queries
   end
 
+  describe "#data_for_expression" do
+    it "when the method has an embedded expression" do
+      yaml_data = <<~YAML
+        ---
+        :db: OrchestrationTemplate
+        :expression:
+          and:
+          - "=":
+              field: OrchestrationTemplate-orderable
+              value: 'true'
+            :token: 1
+          - INCLUDES:
+              field: OrchestrationTemplate-type
+              value: :user_input
+            :token: 2
+      YAML
+
+      f1 = FactoryBot.create(:miq_ae_method,
+                             :name     => "foo_method",
```

Comment:
`name` and `language` both have defaults in the factory and don't look important to the test, should they be left out?